source/gui/nvdaControls.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: UTF-8 -*-
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2016-2018 NV Access Limited
+#Copyright (C) 2016-2018 NV Access Limited, Derek Riemer
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
 import wx
 from wx.lib.mixins import listctrl as listmix
+from gui.accPropServer import *

```

Comment:
I don't think this is good practice right? Can we just import accPropServer instead so that its clear where methods come from?